docs/interacting-with-geth/rpc/ns-debug.md

```diff
@@ -229,6 +229,8 @@ storage hash.
 | Console | `debug.getModifiedAccountsByNumber(startNum uint64, endNum uint64)`             |
 | RPC     | `{"method": "debug_getModifiedAccountsByNumber", "params": [startNum, endNum]}` |
 
+<Note> Geth only keeps a certain number of stateroots in memory - older state roots are deleted by Geth's garbage collection. This means that calls to `debug_GetModifiedAccountsByNumber` on blocks that are old enough to be eligible for garbage collection will return an error due to the state roots being unavailable. To fix this, run Geth with `--cache.preimages=true` to prevent the relevant data being lost to the garbage collector </Note>
```

Comment:
Hmm, the state roots are part of the block headers, its trie nodes and preimages of keys that are deleted for older blocks. The last two sentences are good, only the first one is inaccurate

Reply:
```suggestion
<Note> Geth only keeps recent trie nodes and preimages of keys in memory - for older blocks this information is deleted by Geth's garbage collection. This means that calls to `debug_GetModifiedAccountsByNumber` on blocks that are old enough to be eligible for garbage collection will return an error due to the trie nodes and preimages being unavailable. To fix this, run Geth with `--cache.preimages=true` to prevent the relevant data being lost to the garbage collector </Note>
```